order/interface_test.go

```diff
@@ -190,10 +190,22 @@ func TestOrderReservedValue(t *testing.T) {
 			}
 
 			// For bids the taker pays the most fees if the min
-			// units get matched every block.
+			// units get matched every block. There's an edge case
+			// where if the units unfulfilled is not divisible by
+			// the min units match, then the last match will consume
+			// all the remaining units left.
 			numBlocks := int(o.UnitsUnfulfilled / o.MinUnitsMatch)
-			amt := o.MinUnitsMatch.ToSatoshis()
+			unitsRem := o.UnitsUnfulfilled % o.MinUnitsMatch
+			lastMatch := o.MinUnitsMatch
```

Comment:
Oh, I like this `lastMatch` approach 👍  Maybe easier to read than the `rem` approach I took?